### DISCUSSION TITLE
Turn off loot messages for items covered by other addons

### DISCUSSION THREAD

**Discussion by DakJaniels** [DakJaniels/LuiExtended#232] [Ideas]
![lootlog](https://github.com/ArtOfShred/LuiExtended/assets/107732615/1c3382eb-0665-4686-8291-686929bad0ac)

I'm investigating the possibility of auto turning off certain settings if loot log is enabled. Just another thing to add to my growing idea list. ADHD and coding are the worst mixture lol.

**Comment by ArtOfShred** [comment]
Hmm, personally I prefer LUIE's syntax over Loot Log (thus why I made it in the first place). Having a toggle option might not be bad - do you know if Loot Log uses an inventory index or something to track loot from all sources or does it just use EVENT_LOOT_RECEIVED? (If the latter might be kind of hard to have it toggle off cohesively with LUIE).
